### ISSUE TITLE
Docker run / Podman run commands are not supplied in documentation

### ISSUE THREAD

**Issue by justinkreft** [quipucords/quipudocs#103]
In the old documentation, the docker/podman run commands were supplied for the best way to stand up the containers from the commandline

**Comment by benneteapen** [comment]
I agree. The documentation currently has only instructions for GUI usage. The CLI based instructions are are quite mandatory IMO.
